lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -159,6 +167,10 @@ class EnforceABIImpl {
   std::map<Function *, Function *> OldToNew;
   Function *FunctionDispatcher;
   Function *OpaquePC;
+  DenseMap<std::pair<GlobalVariable *, Function *>, AllocaInst *> CSVMap;
+  DenseMap<Function *, BasicBlock *> UnwindsMap;
+  Function *RaiseExc;
```

Comment:
Let's not have abbreviate names, unless needed. It makes the code harder to read.

Reply:
Second.